ios/Flutter/Zulip.xcconfig

```diff
@@ -0,0 +1,9 @@
+// Configuration settings file format documentation can be found at:
+// https://help.apple.com/xcode/#/dev745c5c974
+
+// TODO(firebase/flutterfire#13323): remove this flag
+//
+// This flag is added to workaround the iOS build failing
```

Comment:
nit: the verb is "work around", and "workaround" is its noun form:
```suggestion
// This flag is added to work around the iOS build failing
```

Just like other verb phrases made of a verb plus preposition, where they're run together to make a noun:
"to log in" / "this login"
"to set up" / "this setup"
"to clean up" / "this cleanup"

(In each case the two versions are pronounced differently, too — the stress moves from the end in the verb form to the beginning in the noun form. That probably contributes to making the difference salient for me.)